back/db/structure.sql

```diff
@@ -1,3 +1,8 @@
+\restrict xUlSfhwHgA9Ktu9jcvzyanAcHX8Oh51d6wjYDKXhantrNfw65o321CZyeDzeami
```

Comment:
This will be annoying for a while (merge conflicts), but it seems like it will be removed in later Rails versions. Otherwise, we could add a "cleaning" part to the rake db tasks that remove this.

Reply:
It's because of a change in PostgreSQL. Rails 8 removes it, and it should have been backported to version 7.2.